ge_getting_started_tutorial/great_expectations/checkpoints/taxi.demo.staging.chk.yml

```diff
@@ -0,0 +1,22 @@
+# This checkpoint was created by the command `great_expectations checkpoint new`.
```

Comment:
This checkpoint and the expectation suite are just direct copies of the versions in ge_demo, so have been "reviewed"/used. Feel free to just skim.